spec/ADAS/ADAS.vspec

```diff
@@ -86,6 +86,18 @@ CruiseControl.IsError:
   type: sensor
   description: Indicates if cruise control system incurred an error condition. True = Error. False = No Error.
 
+CruiseControl.IsAdaptiveActive:
+  datatype: boolean
+  type: actuator
+  description: Indicates if adaptive cruise control system is active (i.e. actively controls speed).
+               True = Active. False = Inactive.
```

Comment:
Do we need to clarify meaning of false? Like what should this signal show if cruise control is active but not in adaptive mode (i.e. it is in old fashioned mode were it keeps the speed until you press the brake pedal or crashes)

Reply:
to my knowledge, there is either adapative or non adaptive cruise control. But it does not change. This signal would just provide the info of what is active in the vehicle...

Reply:
We discussed this further on the Commercial Vehicles' insurance focused breakout call and looked over some of the other nearby (obstacle) signals that exist in production vehicles we would like to see in VSS. Specifically we were looking at left and right blind spot detection mentioned previously and rear cross traffic detection.

ObstacleDetection as it is presently exists in VSS could handle this if there are multiple leaf nodes and well known (defined locations) similar to how multiple tires, seats etc with some generalized optional attributes for 
things like warning type, distance, timegap (desired but debatable).  Vehicle.ADAS.ObstacleDetection.Distance.Forward for example in place of ObstacleDetection.ForwardDistance as was in this PR.

WarningType can have various allowed values such as REAR_CROSS_TRAFFIC so that we know it is detecting a moving instead of stationary object.

Instead of scrapping and proposing a new PR, hope to discuss on an upcoming VSS call as there may be other thoughts on how to represent these.

Reply:
We also decided in CV call that we should focus on mainstream production vehicles with their existing sensors and leave aside more complex configurations of evolving AV/ADAS for the time being. We would be interested in participating in discussion on future proofing VSS signals.